build/package/rpm/go-fdo-server.spec

```diff
@@ -17,7 +19,7 @@ License:        Apache-2.0 AND BSD-3-Clause AND MIT
 URL:            %{gourl}
 Source0:        %{gosource}
 # Generated by go-vendor-tools
-Source1:        %{name}-%{version}-vendor.tar.gz
+Source1:        %{archivename}-vendor.tar.gz
```

Comment:
The current format is intentional as `%{archivename}` does not expand/exist in CentOS Stream 9